### PR TITLE
E2E tests for gov3 and gov4 wallets

### PR DESCRIPTION
**Pull request by rabi-siddique**
refs https://github.com/Agoric/agoric-sdk/issues/10209

This PR introduces end-to-end(E2E) tests that use gov4 to propose questions and then vote on them using both gov4 and gov2 wallets. The objective is to verify whether gov4, as a new member of the EC committee, can effectively execute tasks related to the EC committee and charter.

The PR also adds E2E tests that ensure gov3 can no longer propose and vote on questions.

### CODE REVIEW

**Comment by dckc** [comment]
Let's test that gov3's voting rights are revoked as well.

**Comment by rabi-siddique** [comment]
https://github.com/Agoric/dapp-econ-gov/actions/runs/11550229879/job/32144776496?pr=142

File: .github/workflows/e2e_tests.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@ jobs:
           GH_USERNAME: ${{ secrets.GH_USERNAME }}
           GOV1_PHRASE: ${{ inputs.gov1_mnemonic || secrets.GOV1_PHRASE }}
           GOV2_PHRASE: ${{ inputs.gov2_mnemonic || secrets.GOV2_PHRASE }}
```
Comment (frazarshad) [inline .github/workflows/e2e_tests.yml:96]: not directly related to this PR but we should remove these inputs for gov1 and gov2 and get the phrases from secrets only

File: tests/e2e/utils.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ export const phrasesList = {
     networkConfigURL: 'https://emerynet.agoric.net/network-config',
     gov1Phrase: Cypress.env('GOV1_PHRASE'),
     gov2Phrase: Cypress.env('GOV2_PHRASE'),
+    // UNTIL https://github.com/Agoric/dapp-econ-gov/issues/144
```
Comment (frazarshad) [inline tests/e2e/utils.js:12]: what does this comment mean? i dont think that we will be removing these lines after this issue is resolved

File: tests/e2e/specs/proposal.spec.js [hunk 3]
```diff
@@ -54,7 +65,12 @@ describe('Make Proposal Tests', () => {
       cy.acceptAccess();
     });
 
-    it('should allow gov1 to create a proposal', () => {
+    it('should allow gov2 to create a proposal', () => {
+      if (AGORIC_NET !== 'local') {
```
Comment (frazarshad) [inline tests/e2e/specs/proposal.spec.js:69]: shouldnt this be done for all networks?

File: tests/e2e/utils.js (hunk 2 above)
Comment (rabi-siddique) [reply tests/e2e/utils.js:12]: We won’t remove these lines. Instead, we’ll reverse the addresses. The "UNTIL" comment serves as a reminder and marker for where the update should occur.

File: tests/e2e/specs/proposal.spec.js (hunk 3 above)
Comment (rabi-siddique) [reply tests/e2e/specs/proposal.spec.js]: I think so. Let me verify

File: tests/e2e/utils.js (hunk 2 above)
Comment (frazarshad) [reply tests/e2e/utils.js:12]: oh. missed that. thanks for clarifying

**Review by frazarshad — APPROVED** [review]
LGTM

**Pull request merged**